package.json

```diff
@@ -12,7 +12,7 @@
     "npm": ">=6.4.1"
   },
   "scripts": {
-    "test": "grunt test",
+    "test": "grunt eslint && grunt unit && grunt test-api-integration && grunt e2e",
```

Comment:
eslint as well because I'm using this as a "have I messed anything up" mega action